specificatie/openapi.yaml

```diff
@@ -261,6 +482,13 @@ components:
         - properties:
             _links:
               $ref: "#/components/schemas/VerblijfplaatshistorieLinks"
+    VerblijfplaatshistorieCollectie:
+      type: object
+      properties:
+        verblijfplaatshistorie:
+          type: array
+          items:
+            $ref: '#/components/schemas/Verblijfplaatshistorie'
     Verblijfplaatshistorie:
       allOf:
       - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/v1.2.0/specificatie/openapi.yaml#/components/schemas/Verblijfplaats"
```

Comment:
wellicht kan het nu nog niet, maar straks moet dit naar v2.0.0 van BRP-bevragen verwijzen. Nu al is de actuele versie v1.3.0

Reply:
No naar 1.3.0 laten verwijzen. Verwijzen naar v2.0.0 kan pas als die release ook gemaakt is in github.